README.md

```diff
@@ -134,6 +134,7 @@ Express middleware supports the following properties:
 + `options`
   + `endpointUrl` [`string`] - the GraphQL endpoint url.
   + `displayOptions` [`object`] - same as [here](#Properties)
+  + `headersJS` [`string`, default `"{}"`] - object of headers serialized in string to be used on endpoint url
```

Comment:
@stekycz Would be  great if you add a note that JS can be used there, e.g.:
> You can also use any JS expression which results in an object with header names as keys and strings as values e.g.  `{ Authorization: localStorage['Meteor.loginToken'] }`

Everything else is great 👍

Reply:
@IvanGoncharov Updated